modules/swagger-codegen/src/main/resources/swift/AlamofireImplementations.mustache

```diff
@@ -96,7 +96,7 @@ class AlamofireRequestBuilder<T>: RequestBuilder<T> {
                 if (dataResponse.result.isFailure) {
                     completion(
                         response: nil,
-                        error: dataResponse.result.error
+                        error: ErrorResponse.Error(response.response?.statusCode ?? 500, response.data, response.result.error!)
```

Comment:
This should be `dataResponse`. Could you try running `./bin/swift-petstore.sh && mvn verify -Pswift-client`?